tt_metal/host_api.hpp

```diff
@@ -109,6 +109,8 @@ bool CloseDevice(Device *device);
  */
 Program CreateProgram();
 
+Program CreatePersistentProgram();
```

Comment:
You mentioned in the PR description that Persistent Program has a limitation related to CB attachment.
If I understood that right, such a limitation has to be reflected in the type of program to avoid mistakes.
Wdyt?